modin/config/envvars.py

```diff
@@ -848,4 +848,22 @@ def _check_vars() -> None:
         )
 
 
+class NativeDataframeMode(EnvironmentVariable, type=str):
+    """
+    The mode of execution used for handling dataframes in Modin.
+
+    When the env variable is set to None the PandasQueryCompiler would be used
+    which would lead to Modin executing dataframes in distributed fashion.
+    When set to Native_pandas NativeQueryCompiler is used which handles the
+    dataframes without distributing, falling back to native pandas functions.
+
+    In future more execution modes can be added for single node execution so
+    keeping the parameter as string.
+    """
+
+    varname = "MODIN_NATIVE_DATAFRAME_MODE"
+    choices = ("Native_pandas",)
+    default = None
```

Comment:
```suggestion
    default = "Modin"
```
What do you think on default value "Modin"?

Reply:
I will rename it to Modin, then I think it would be better to rename the varname from MODIN_NATIVE_DATAFRAME_MODE -> NATIVE_DATAFRAME_MODE as having MODIN in the varaible name and setting it to Modin for the default execution seems a bit off.

Reply:
Then still should we set the default value to "Modin" can we leave it as None, which would mean no NATIVE_DATAFRAME_MODE is chosen, another options could be "Distributed" or "Default"

Reply:
Changed